docs/hyperion-setup.md

```diff
@@ -1 +1,411 @@
+
+The plugin manager requires a configuration. You can see the full set of options [here](/pluginmanager/README.md), but for now lets start with the bare basics. Put the following in `pluginmanager.yml`:
+
+```yaml
+host: "tcp://localhost:3000"
+plugins: []
+```
+
+We don't have any steps in our pipeline yet, but we will be adding those as we continue the tutorial. For now, you should be able to run the following command to start the plugin manager:
+
+```shell script
+$ java -jar path/to/pluginmanager-all.jar pluginmanager.yml
+```
+
+### Step 1: Sourcin' the data
+
+Let's start by first ensuring that the pipeline is able to actually retrieve our data. Since we're using the ELK stack, we actually have two options: using the [elasticsearch data source](/datasource/plugins/elasticsearch) or using the [logstash-output-hyperion](/logstash-output-hyperion) plugin. For the sake of simplicity, we will be using the elasticsearch data source in this tutorial, but for production environments you should be using the Logstash plugin, as it is far more performant.
+
+Assuming that your elasticsearch instance is running on localhost and requires no auth, we can use the following bare configuration for the data source (a full list of options can be found [here](/datasource/plugins/elasticsearch/README.md)). Put the following in `elasticsearch.yml`:
+
+```yaml
+poll-interval: 5
+
+elasticsearch:
+    hostname: localhost
+    port: 9200
+    index: logs  
+    scheme: http
+    timestamp-field: "@timestamp"
+    authentication: false
+    response-hit-count: 10000
+
+pipeline:
+    manager-host: "localhost:3000"
+    plugin-id: "ElasticSearch"
+```
+
+As you can see, we have configured the pipeline manager host to point to the port 3000 that we configured in step 0. If we were to start this plugin right now though, it'd error. Why? Because the plugin manager doesn't know anything about the `ElasticSearch` plugin yet and therefore can't tell it where the data needs to go. Let's fix that first. Edit your `pluginmanager.yml` such that instead of it having an empty `plugins` section, it instead reads the following:
+
+```yaml
+plugins:
+  - id: "ElasticSearch"
+    host: "tcp://localhost:3001"
+```
+
+Make sure to restart your plugin manager, then start the elastic search data source as follows:
+
+```shell script
+$ java -jar path/to/elasticsearch-all.jar run elasticsearch.yml
+```
+
+If everything goes according to plan, the ElasticSearch data source will start broadcasting log data on port 3001. Now, let's add some plugins.
+
+### Step pre-2: What data are we actually processing?
```

Comment:
I think the title could be changed to something like "sidenote" instead of a pre-step